cmd/gosky/main.go

```diff
@@ -1469,3 +1470,72 @@ var createFeedGeneratorCmd = &cli.Command{
 		return nil
 	},
 }
+
+var blocksCmd = &cli.Command{
+	Name: "blocks",
+	Subcommands: []*cli.Command{
+		blocksAddCmd,
+		blocksListCmd,
+	},
+}
+
+var blocksAddCmd = &cli.Command{
+	Name:      "add",
+	Flags:     []cli.Flag{},
+	ArgsUsage: `<user>`,
+	Action: func(cctx *cli.Context) error {
+		xrpcc, err := cliutil.GetXrpcClient(cctx, true)
+		if err != nil {
+			return err
+		}
+
+		user := cctx.Args().First()
```

Comment:
we might want to confirm that this is a DID instead of a handle, or if its a handle resolve it to a DID first